modules/sops/default.nix

```diff
@@ -70,6 +70,14 @@ let
           Sops file the secret is loaded from.
         '';
       };
+      sopsFileHash = mkOption {
+        type = types.str;
+        readOnly = true;
+        default = if cfg.validateSopsFiles then "${builtins.hashFile "sha256" config.sopsFile}" else "";
```

Comment:
Is this different from using `config.sopsFile` directly?

Reply:
`config.sopsFile` might not exist at evaluation time, and can cause evaluation failures.

Reply:
The file path is within the flake source directory in nix store, resulting in unnecessary restarts of services.